test/scripts/test_neighbors.py

```diff
@@ -28,5 +31,5 @@ def test_neighbors_get(self, ser: SerialHelper):
 
             # Ensure that a neighbor node ID exists and the port found
             # matches the port expected number
-            assert isinstance(neighbor_info[0], int)
-            assert neighbor_info[1] == port
+            if not isinstance(neighbor_info[0], int) or neighbor_info[1] != port:
+                pytest.fail("Failed neighbor test and cannot move on")
```

Comment:
If these do the same thing, I think the original 2 assertions are easier to understand. Does this change accomplish something else?

Reply:
The reason for this change was to end all tests if this test cannot be completed. Mainly because all other tests rely on neighbor functionality working (mainly because most other tests rely on topology to work which relies on neighbors). This will fail all tests and output to the user why.

Reply:
💡 Ah! Got it. Thanks, good call, keep it. 👍